presto-raptor/src/main/java/com/facebook/presto/raptor/storage/BucketBalancer.java

```diff
@@ -153,22 +153,30 @@ public CounterStat getJobErrors()
         return jobErrors;
     }
 
-    private void startJob()
+    private void scheduleBalanceJob()
```

Comment:
Let's just inline this method since it's now one line and the class only has one job